CS410-Assignment2/src/sensorApplication/SensorApplication.java

```diff
@@ -0,0 +1,63 @@
+package sensorApplication;
+
+import java.awt.Color;	//	bar color
+import java.awt.Dimension;
+import java.awt.GridLayout;
+
+import javax.swing.JFrame;	//	used by SensorApplication
+import javax.swing.JLabel;	//	sensor readouts
+import javax.swing.JPanel;	//	used by SensorPanel
+import javax.swing.border.TitledBorder;	//	used for panel border names
+import javax.swing.JProgressBar;	//	used for bar levels
+
+import sensorAdapter.*;
+import sensor.*;
+
+//	Adapted from example code
+public class SensorApplication extends JFrame {
+	
+	//	JPanel used by each sensor
+	public class SensorPanel extends JPanel {

```

Comment:
It seems to me that this class would have been better to put in another file, although it's not very large and is suited for the JFrame directly. If it is going to be here, shouldn't it be private?

Reply:
Good point, it should be private.